api/tests/Api/CampCollaborations/CreateCampCollaborationTest.php

```diff
@@ -369,6 +369,52 @@ public function testCreateCampCollaborationValidatesMissingUserAndInviteEmail()
         ]);
     }
 
+    public function testCreateCampCollaborationValidatesMissingUserAndInviteEmailAndIncludesTranslationInfo() {
+        static::createClientWithCredentials()->request('POST', '/camp_collaborations', ['json' => $this->getExampleWritePayload([], ['user'])]);
+
+        $this->assertResponseStatusCodeSame(422);
+        $this->assertJsonContains([
+            'violations' => [
+                [
+                    'i18n' => [
+                        'key' => 'app.validator.asserteitherisnull',
+                        'parameters' => [
+                            '{{ other }}' => 'user',
```

Comment:
Do we really want the brackets?

Or would we better have it without:
`'other' => 'user'`

2nd might be easier to process by the client.

Reply:
I cannot derive the double curly braces from the ICU Message format, only single curly brace.
The double curly braces would occur with pluralization, but not with normal parameters.
-> use no curly braces at all.
Implemented with: https://github.com/ecamp/ecamp3/pull/2953/commits/7819f7b319ff788b1409514da1b02e6fd9ce40b5 and https://github.com/ecamp/ecamp3/pull/2953/commits/190922df746c886226fdd80b6bf053a77bcfbb1d